packages/xo-server/src/api/host.js

```diff
@@ -215,6 +215,20 @@ emergencyShutdownHost.resolve = {
 
 // -------------------------------------------------------------------
 
+export function assertConsistentHostServerTime({ host }) {
```

Comment:
For me `assert` (`check` to a lesser extent) means that it will throws if the condition is not fulfilled.

IMHO, if it returns a boolean, it should be called `isConsistentHostServerTime`.

Reply:
Ok so my (unclear) question should have been: should this return a boolean (and be named accordingly)?

Reply:
Yes, for an API, I agree it makes more sense.